PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalClient.kt

```diff
@@ -304,7 +310,8 @@ class PayPalClient internal constructor(
         braintreeClient.sendAnalyticsEvent(PayPalAnalytics.TOKENIZATION_FAILED, analyticsParams)
 
         if (isAppSwitchFlow) {
-            braintreeClient.sendAnalyticsEvent(PayPalAnalytics.APP_SWITCH_FAILED, analyticsParams)
+            val appSwitchAnalyticsParams = analyticsParams.apply { appSwitchUrl = appSwitchUrlString }
```

Comment:
🤔 it looks like that doesn't set the property since `analyticsParams` is a `val`. Changing it to a `var` gives some compiler warnings and updating to the suggestion doesn't seem to be setting the other properties. Likely just my misunderstanding on the best way to do this since I agree it feels a bit ugly as it is in the PR.

Reply:
Ah missed that - this way is good then 👍

Reply:
Oh nice! Updated to use copy here: https://github.com/braintree/braintree_android/pull/1226/commits/45c98f720d4a0fc58c23e5f142cf12046c4f1d53